client/my-sites/sidebar-unified/index.jsx

```diff
@@ -48,34 +50,39 @@ export const MySitesSidebarUnified = ( { path } ) => {
 	}
 
 	return (
-		<Sidebar>
-			<li>
-				<CurrentSite forceAllSitesView={ isAllDomainsView } />
-			</li>
-			{ menuItems.map( ( item, i ) => {
-				const isSelected = item?.url && itemLinkMatches( item.url, path );
+		<>
```

Comment:
would you mind using `<Fragment>` instead of `<>`?

Reply:
Or we could just nest `<AsyncLoad require="calypso/blocks/nav-unification-modal" placeholder={ null } />` in `<Sidebar>`. But I reckon that this is not semantically ideal since the modal doesn't render inside the sidebar.

Reply:
Nesting `<AsyncLoad />` doesn't feel right, so I've updated to use the Fragment component instead :)